config/backend-config/multitenant_workspaces_test.go

```diff
@@ -126,7 +126,9 @@ func TestMultiTenantWorkspacesConfig_Get(t *testing.T) {
 		}
 		conf, err := wc.Get(context.Background(), "")
 		require.NoError(t, err)
-		require.Equal(t, sampleBackendConfig, conf)
+		multiTenantWorkspacesConfig := sampleBackendConfig
+		multiTenantWorkspacesConfig.ConnectionFlags = ConnectionFlags{URL: "", Services: map[string]bool{"warehouse": true}}
```

Comment:
You could do a `t.Setenv("CP_ROUTER_URL")` and verify that the `ConnectionFlags.URL` gets that value instead of just `""`.